### DISCUSSION TITLE
Adjusting the Thickness of the Donut Chart

### DISCUSSION THREAD

**Discussion by mikereilly13** [tremorlabs/tremor-npm#570] [Q&A]
Is it possible to increase the thickness of the donut chart?

**Comment by AikoRamalho** [comment]
Any thoughts on this? @severinlandolt  @christopherkindl @mitrotasios 

I want to increase the thickness of my donut component as well and haven't found any reference about this in the docs

**Reply by severinlandolt** [reply]
Hi @AikoRamalho, adjusting the outer and inner radii is not possible. Will discuss this and maybe convert it to an issue.
